### PR TITLE
[TF-TRT] Fix an uninitialized variable in ConvertEinsum.

### PR DESCRIPTION
**Pull request by christopherbate**
Uninitialized variable `output_has_ellipsis` would periodically result in test failure due to undefined behavior. `EinsumHelper::ParseEquation` apparently does not set this variable under certain parsing conditions.

Removed `#if` guard on test as test failure seems unrelated to TRT8.

Formatting applied after `#if` guard removed.

@tfeher 
@bixia1

### CODE REVIEW

File: tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc [hunk 1]
```diff
@@ -5783,7 +5783,7 @@ Status ParseEquation(OpConverterParams* params,
   OperandLabelCounts input_label_counts;
   LabelCounts output_label_counts;
   absl::InlinedVector<bool, 2> input_has_ellipsis;
```
Comment (tfeher) [inline tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:5785]: Could you initialize these too? [ParseEquation](https://github.com/tensorflow/tensorflow/blob/b47357628fa624676db35186d57b416462f8bcd5/tensorflow/core/kernels/linalg/einsum_op_impl.h#L160) sets also those conditionally, and we use them below.

**Review by tfeher — COMMENTED** [review]
Thanks at @christopherbate for catching this problem! Just one request, otherwise looks good to me.

File: tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc (hunk 1 above)
Comment (christopherbate) [reply tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:5785]: These are all inlined vector which are empty as declared here. The template parameter is just a capacity if I understand correctly.

File: tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc [hunk 2]
```diff
@@ -5783,7 +5783,7 @@ Status ParseEquation(OpConverterParams* params,
...
-  bool output_has_ellipsis;
+  bool output_has_ellipsis{false};
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc:5786]: Good catch! It seems to me that there are similar problem with input_has_ellipsis. Checking with TensorFlow people on this.

**Comment by bixia1** [comment]
Thanks for catching the unitialization bug in the Einsum parser. Since input_has_ellipsis also has the bug, I am working on the fix in the parser itself  (change submitted for review within google).

You have this in your PR description:
Removed #if guard on test as test failure seems unrelated to TRT8.

Your change enables the test for TRT8. Do you actually mean the test passes with TRT8? If that is the case, can you revise the PR title and the PR description so that I can approve the change for the test?

**Comment by christopherbate** [comment]
I think the bug is fixed. Let me double-check as this was a week ago.

**Comment by mihaimaruseac** [comment]
Is this still an issue? I think this should be fixed.

**Pull request closed without merging**